runtime/vm/MHInterpreter.inc

```diff
@@ -19,6 +19,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
+#if defined(J9VM_OPT_METHOD_HANDLE)
```

Comment:
To be strictly correct, the ifdef should appear after including j9cfg.h.

Reply:
More than that, it's just wrong before that include (the symbol will always be undefined).

Reply:
This isn't a real file - it's included from a .cpp that almost certainly already has included j9cfg.h.

Reply:
Perhaps, but better safe than sorry.